src/utils/pickupLocationsUtils.ts

```diff
@@ -0,0 +1,50 @@
+import { type SierraCodeName } from "../types/myAccountTypes"
+
+const cache = {
```

Comment:
I know these functions are mostly self-explanatory, but I think it would be good to keep adding comments with a brief description.

Reply:
i added some comments